src/reducers/clusterReducer.js

```diff
@@ -150,6 +150,50 @@ export default function clusterReducer(
         items: items,
       };
 
+    case types.CLUSTER_LOAD_APPS:
+      items = Object.assign({}, state.items);
+
+      items[action.clusterId] = Object.assign({}, items[action.clusterId], {
+        isFetchingApps: true,
+      });
+
+      return {
+        lastUpdated: Date.now(),
+        isFetching: false,
```

Comment:
Is that right? Do we use two different flags for fetching?

Reply:
It's a bit confusing, but `isFetching` is on the collection of `clusters`. So that is true when we're doing `/v4/clusters/` and `isFetchingApps` is on a item in that collection (aka a cluster). So that is true/false depending on what cluster we're looking at, and is related to the `/v4/clusters/:clusterid/apps/` call.